src/components/Chat/ModalImageSingleDetail.js

```diff
@@ -42,7 +42,7 @@ const ModalImageSingleDetail = ({onBack, visible, img, name, time}) => {
             <MemoIc_arrow_turn_right width={20} height={20} />
           </TouchableOpacity>
           <TouchableOpacity>
-            <Icon name="dots-three-vertical" color={COLORS.white} size={17} />
+            <Icon name="dots-three-vertical" color={COLORS.almostBlack} size={17} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the "dots-three-vertical" icon has been changed from white to almost black. Ensure that this color change is consistent with your application's theme and provides sufficient contrast for visibility on all screens where it is used.
```diff
- <Icon name="dots-three-vertical" color={COLORS.white} size={17} />
+ <Icon name="dots-three-vertical" color={COLORS.almostBlack} size={17} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->